tests/snapshots/execution_spec__test-grpc-invalid-proto-id.md_errors.snap

```diff
@@ -4,7 +4,7 @@ expression: errors
 ---
 [
   {
-    "message": "File descriptor not found for method: abc.NewsService.GetAllNews",
+    "message": "Protobuf files were not specified in the config",
```

Comment:
I think this should be validated at compile time.

Reply:
this is validation error snapshot for compilation stage